src/lib/holocene/tooltip.svelte

```diff
@@ -54,7 +60,7 @@
 
 <style lang="postcss">
   .tooltip {
-    @apply invisible absolute left-0 top-0 z-50 inline-block translate-x-12 whitespace-nowrap text-xs opacity-0 shadow-md transition-all;
```

Comment:
I've removed the `shadow-md` as it was causing this behavior 👇 

| Existing | Updated|
| ------- | -------|
|![Screen Shot 2022-08-01 at 4 31 55 PM](https://user-images.githubusercontent.com/15069288/182436244-600ae9c5-ea69-4f9c-bc7a-2d7970e64400.png)|![Screen Shot 2022-08-01 at 4 32 20 PM](https://user-images.githubusercontent.com/15069288/182436269-1c8a3a18-0288-4fe8-8d15-1ea5579e021c.png)|

Reply:
Good catch!